README.md

```diff
@@ -3,10 +3,18 @@
 [![CircleCI](https://circleci.com/gh/DataBiosphere/data-explorer.svg?style=svg)](https://circleci.com/gh/DataBiosphere/data-explorer)
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-## Quickstart
+## Overview
+
+Data Explorer lets you explore a dataset. The code (in this repo and
+[data-explorer-indexers repo](https://github.com/DataBiosphere/data-explorer-indexers))
+is dataset-agnostic. All dataset configuration happens in config files.
 
-View [Data Explorer](https://test-data-explorer.appspot.com/) for the public
-[1000 Genomes](http://www.internationalgenome.org/about) dataset.
+For example, [here's Data Explorer](https://test-data-explorer.appspot.com/)
```

Comment:
tiny nit: "here's **the** Data Explorer"?

Reply:
Done